### PR TITLE
aesthetics update

### PR DESCRIPTION
**Pull request by SRich-ing**
Colours and style updated

### CODE REVIEW

File: src/data/db.py [hunk 1]
```diff
@@ -7,9 +7,9 @@ class DB:
     def __init__(self, connection=None):
         self.host = os.getenv("DB_HOST")
         self.port = int(os.getenv("DB_PORT", 3306))
-        self.database = os.getenv("DB_NAME")
-        self.user = os.getenv("DB_USER")
-        self.password = os.getenv("DB_PASSWORD")
+        self.database = 'uni_app'
```
Comment (mariatzele) [inline src/data/db.py]: You cannot change these here. We don't all have the same password for the database. You should create a `.env` file as described in the README.md

**Review by mariatzele — CHANGES_REQUESTED** [review]
Please see my notes. Do not merge without removing the changes in `db.py` - it will break the app

File: src/data/db.py (hunk 1 above)
Comment (sgtho) [reply src/data/db.py]: @SRich-ing Is that possible to create a database by following the README from your side? If so, we should all do that to make sure the process is suitable for everyone.

Comment (SRich-ing) [reply src/data/db.py]: Agreed - left that in by mistake!

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Review by sgtho — APPROVED** [review]

**Pull request merged**